data/map.txt

```diff
@@ -28304,6 +32227,14 @@ planet "Aresepru Nat"
 	landscape land/sea0
 	description `Now that the Korath are gone, the oceans on this planet have begun to teem with fish and other living creatures once again, and the levels of dissolved oxygen and carbon dioxide are slowly swinging back into balance. Species of algae and bacteria that are able to metabolize the plastics and chemicals that the Korath left behind have spread throughout the oceans.`
 	security 0
+	
+planet Arroharg
+	landscape land/canyon12
+	description `Arroharg is a low-gravity moon that the Quarg have converted to some sort of harvesting grounds. Large organic-looking machines seem to be displacing particular mountainous formations while smaller ones collect from the zones left behind. Most of the architecture here is Quarg-like and is not designed in a way that welcomes visitors very comfortably. The lower levels of the outpost seem to dig deeper into the surface of the moon, but the farther they go down the more complex they become. There's no telling how deep into the moon they go, as they are restricted to anyone but the Quarg.`
+	spaceport `A few Quarg greet you with astonished expressions but kindly assist you in anything you have trouble with due to the spaceport being designed around their architecture. They communicate mostly in gestures and facial expressions, although this is sometimes hard to decifer. It seems they aren't used to outside visitors here and don't speak your language.` 
```

Comment:
Any reason why this was not resolved? I think github is bugging out, I swore I already commented this change like twice before.

Reply:
It's because here, I manually resolved the conversation, and manually changed it in the files. I'll go double-check, but this looks like a visual bug on Git. I've had to do this with your map and missions reviews because they were doing the diff thing again. I think it's because I did do those pushes yesterday.

Reply:
Ah, that's weird. Thanks!